HLTrigger/Configuration/python/Eras.py

```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
+
+# import the relevant eras from Configuration.Eras.*
+from Configuration.Eras.Modifier_trackingPhase1_cff import trackingPhase1
+
+def modifyHLTforEras(fragment):
+    """load all Eras-based customisations for the HLT configuration"""
+
+    # modify the HLT configuration for the Phase I pixel geometry
+    from HLTrigger.Configuration.customizeHLTTrackingForPhaseI2017 import customizeHLTPhaseIPixelGeom
+    trackingPhase1.toModify(fragment, customizeHLTPhaseIPixelGeom)
+
+    # modify the HLT configuration to run the Phase I tracking in the particle flow sequence
+    from HLTrigger.Configuration.customizeHLTTrackingForPhaseI2017 import customizeHLTForPFTrackingPhaseI2017
+    trackingPhase1.toModify(fragment, customizeHLTForPFTrackingPhaseI2017)
```

Comment:
It's essentially about having multiple tracking configurations integrated and maintained (current phase1 use cases are mainly comparison reference purposes).

* `Run2_2017` is the default being run in RelVals (could be called "production version").
* `Run2_2017_trackingRun2` runs the 2016 tracking on phase1 detector (works technically even if the 4th layer is not used in seeding).
* `Run2_2017_trackingPhase1PU70` has a tracking close to the phase1 TDR (or phase2 TP PU70)

#16911 will (after an update coming later today) remove `Run2_2017_trackingPhase1PU70` (*) and add `Run2_2017_trackingPhase1CA` (**).

(*) TDR tracking was useful during the year but is not needed longer
(**) later CA seeding will be made the default, and what we have now in `Run2_2017` will be put to some `Run2_2017_trackingX` era.